packages/node/src/machine/types.ts

```diff
@@ -43,6 +44,12 @@ export type ProtocolMessage = {
   customData: { [key: string]: any };
 };
 
+export type FinMessage = {
+  toXpub: string;
+  processID: string;
+  eventName: Node.EventName;
```

Comment:
should this be the event name or the protocol name?

Reply:
Event name. We don't care about the protocol name here because the protocol for which this event is being sent around is inherently linked to the `processID`.